src/extension/tab/hook.ts

```diff
@@ -117,18 +117,9 @@ function getClientData() {
 
 handleRpc("getClientOperations", getClientData);
 
-function sendHookDataToDevTools(eventName: "connectToDevtools") {
-  tab.send({
-    type: eventName,
-    payload: getClientData(),
-  });
-}
-
 tab.on("connectToClient", () => {
   if (hook.ApolloClient) {
-    sendHookDataToDevTools("connectToDevtools");
-  } else {
-    findClient();
```

Comment:
The devtools script will now try to connect and then wait up to 10 seconds for a `registerClient` or `connectToDevtools` message before giving up and showing the not found modal. Since we can rely on those messages instead, we don't need to try and find a client when this `connectToClient` message is sent.

In the future, this `connectToClient` could likely be replaced with an RPC call to simplify this a bit further. This adds a bit of complexity though to do this in this PR so I'll do this separately to keep this PR clean.